content/en/docs/Overview/_index.md

```diff
@@ -18,14 +18,14 @@ The parquet-format repository hosts the official specification of the Apache Par
 As a repository focused on specification, the parquet-format repository does not contain source code. 
 
 
-### parquet-mr
+### parquet-java
```

Comment:
maybe note here that this was previously referred to as parquet-mr due to the name of the repository (which has also been moved)?

Reply:
i agree adding a note like this would be clearer

```
The parquet-java repository(previously named `parquet-mr`) is part of the Apache Parquet project and specifically focuses on providing Java tools for
```